doc/developer/packaging-redhat.rst

```diff
@@ -89,7 +95,7 @@ Tested on CentOS 6, CentOS 7, CentOS 8 and Fedora 24.
       %{!?with_pimd:          %global  with_pimd          1 }
       %{!?with_rpki:          %global  with_rpki          0 }
 
-8. Build the RPM::
+9. Build the RPM::
 
       rpmbuild --define "_topdir `pwd`/rpmbuild" -ba rpmbuild/SPECS/frr.spec
```

Comment:
Following the instructions exactly on a newly installed centos 7 vm I get:
     rpmbuild --define "_topdir `pwd`/rpmbuild" -ba rpmbuild/SPECS/frr.spec
error: Failed build dependencies:
        libyang-devel >= 1.0.184 is needed by frr-7.7_dev-01.el7.x86_64

output from make install of libyang:
Install the project...
-- Install configuration: "Release"
-- Up-to-date: /usr/lib64/libyang.so.1.10.7
-- Up-to-date: /usr/lib64/libyang.so.1
-- Up-to-date: /usr/lib64/libyang.so
-- Up-to-date: /usr/include/libyang/tree_schema.h
-- Up-to-date: /usr/include/libyang/tree_data.h
-- Up-to-date: /usr/include/libyang/extensions.h
-- Up-to-date: /usr/include/libyang/user_types.h
-- Up-to-date: /usr/include/libyang/xml.h
-- Up-to-date: /usr/include/libyang/dict.h
-- Up-to-date: /usr/include/libyang/libyang.h
-- Up-to-date: /usr/lib64/pkgconfig/libyang.pc
-- Up-to-date: /usr/bin/yanglint
-- Up-to-date: /usr/share/man/man1/yanglint.1
-- Up-to-date: /usr/bin/yangre
-- Up-to-date: /usr/share/man/man1/yangre.1
-- Up-to-date: /usr/lib64/libyang1/extensions/nacm.so
-- Up-to-date: /usr/lib64/libyang1/extensions/metadata.so
-- Up-to-date: /usr/lib64/libyang1/extensions/yangdata.so
-- Up-to-date: /usr/lib64/libyang1/user_types/user_yang_types.so
-- Up-to-date: /usr/lib64/libyang1/user_types/user_inet_types.so

Reply:
The Libyang dependency should be resolved from Step 2 in this file "Install build dependency" which I read as refering to the other doc with all the build packages